package cn.tzs.exception;

public class TestException1 {
	public static void main(String[] args){
		
	}
}
